doc/sphinx/source/users-guide/locality/onClauses.rst

```diff
@@ -0,0 +1,111 @@
+:title: Users Guide: On-Clauses
+
+.. default-domain:: chpl
+
+On-Clauses
+==========
+
+As mentioned previously, all Chapel programs begin execution as a
+single task running on locale #0.  Tasks can execute on remote locales
```

Comment:
It's not necessarily a remote locale, is it?  We can use an on-clause to migrate to a sublocale of the current locale. I think you could reasonably replace this sentence with something like "Execution can be moved using on-clauses."